language/predefined/typeerror.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- $Revision$ -->
+<!-- EN-Revision: 3fc56d76de1007a3c4f60b143c60c071f32546fb Maintainer: samesch Status: ready -->
+<!-- Reviewed: no -->
+<phpdoc:exceptionref xml:id="class.typeerror" xmlns:phpdoc="http://php.net/ns/phpdoc" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xi="http://www.w3.org/2001/XInclude">
+ <title>TypeError</title>
+ <titleabbrev>TypeError</titleabbrev>
+
+ <partintro>
+
+<!-- {{{ Error intro -->
+  <section xml:id="typeerror.intro">
+   &reftitle.intro;
+   <para>
+    Ein <classname>TypeError</classname> kann ausgelöst werden, wenn:
+    <simplelist>
+     <member>
+      Der Wert, der für eine Klasseneigenschaft festgelegt wird, entspricht
+      nicht dem deklarierten Typ der Eigenschaft.
```

Comment:
Sounds better with the Introduction above. 

(Same for the other members under this suggestion). 

```suggestion
      Der Wert, der für eine Klasseneigenschaft festgelegt wird,
      nicht dem deklarierten Typ der Eigenschaft entspricht.
```